sdcm/sct_events.py

```diff
@@ -185,7 +185,7 @@ def formatted_timestamp(self):
 
     def publish(self, guaranteed=True):
         if guaranteed:
-            return EVENTS_PROCESSES['MainDevice'].publish_event_guaranteed(self)
+            return EVENTS_PROCESSES['MainDevice'].publish_event_guaranteed(self, reuse_socket=False)
```

Comment:
this issue was addressed in #1757

Reply:
I know, but I'm trying to get this branch working.
and it keep failing on this issue, I'll remove it before merging.